pyof/foundation/network_types.py

```diff
@@ -6,9 +6,82 @@
 from pyof.foundation.base import GenericStruct
 from pyof.foundation.basic_types import (
     BinaryData, HWAddress, IPAddress, UBInt8, UBInt16)
+from pyof.foundation.constants import VLAN_TPID
 from pyof.foundation.exceptions import PackException
 
-__all__ = ('Ethernet', 'GenericTLV', 'IPv4', 'TLVWithSubType', 'LLDP')
+__all__ = ('Ethernet', 'GenericTLV', 'IPv4', 'VLAN', 'TLVWithSubType', 'LLDP')
+
+
+class VLAN(GenericStruct):
+    """802.1q VLAN header."""
+
+    #: tpid (:class:`UBInt16`): Tag Protocol Identifier
+    tpid = UBInt16(VLAN_TPID)
+    #: _tci (:class:`UBInt16`): Tag Control Information - has the
+    #: Priority Code Point, DEI/CFI bit and the VLAN ID
+    _tci = UBInt16()
+
+    def __init__(self, pcp=None, cfi=None, vid=None):
+        """The constructor receives the parameters below.
+
```

Comment:
@renanrodrigo please, add a comment stating that if no arguments are passed and/or set and the pack is called, then the pack will return an empty binary value, meaning that there is no VLAN information. [You can do a better description than mine, but I think you've got the idea]